MoreLinq/EquiZip.cs

```diff
@@ -49,11 +51,7 @@ static partial class MoreEnumerable
         /// "2B", "3C", "4D" in turn.
         /// </example>
         /// <remarks>
-        /// <para>
```

Comment:
This should be moved to into an `<exception>` section following the `<returns>` section, like to:

```c#
        /// <returns>
        /// A sequence that contains elements of the two input sequences,
        /// combined by <paramref name="resultSelector"/>.
        /// </returns>
        /// <exception cref="InvalidOperationException">
        /// The input sequences are of different lengths.</exception>
        /// <example>
```